bundle/webpack.config.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
 			svgs: path.join(__dirname, 'static', 'svg'),
 			'ophan/ng': 'ophan-tracker-js',
 			lodash: 'lodash-es',
+			process: 'process/browser',
```

Comment:
Just for my understanding why do we need this in the prod build if it wasn't used in Frontend?

Reply:
Great point
Looking at frontend's webpack it's got:
https://github.com/guardian/frontend/blob/main/webpack.config.prod.js#L31-L33

That would make sure process is defined I think
And would be better because the rest of `process.*` won't be in the bundle